cpp/include/Ice/Object.h

```diff
@@ -31,6 +33,24 @@ namespace Ice
     public:
         virtual ~Object() = default;
 
+        /**
+         * Dispatch an incoming request and return the corresponding outgoing response.
+         * @param request The incoming request.
+         * @param sendResponse A callback that the implementation calls to return the response. sendResponse does not
+         * throw any exception and any sendResponse wrapper must not throw any exception. sendResponse can be called by
```

Comment:
With connections, it messes up with the dispatch count of the connection. Presumably, the result is the connection shutdown wouldn't wait for all outstanding dispatches (since the dispatch count is too low). I could not think of a simple way to describe this situation.